src/pages/BirthDay.jsx

```diff
@@ -1,11 +1,48 @@
 import Layout from "../components/Layout/Layout";
+import CakeIcon from "@mui/icons-material/Cake";
+import { styled, Stack, Typography } from "@mui/material";
```

Comment:
```suggestion
import CakeIcon from "@mui/icons-material/Cake";
import { styled, Stack, Typography } from "@mui/material";

import Layout from "../components/Layout/Layout";
```

import 문은 항상 개행으로 구분하도록